stdlib/public/core/Array.swift

```diff
@@ -1800,6 +1800,9 @@ extension Array {
   ///     }
   ///     // numbers == [1, 2]
   ///
+  /// - Note: This example shows the behavior on a [little-endian][1] platform.
```

Comment:
Adding a blank line here should the note and link from interfering with each other.

(I'm not 100% sure of the behavior in the markdown dialect that's used to parse these, and there isn't a good way to test ahead of time, so it's better be safe.)

```suggestion
  /// - Note: This example shows the behavior on a [little-endian][1] platform.
  ///
```

Reply:
fixed